### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by williamrolando88**
Hi, in this PR, I:
- Installed the required dependencies for the covid tracker project
- Setup the storage with redux-toolkit
- Fetch information from the external database and store it into redux storage
- Created the required components i.e. HomePage, Country
- Setup the routes management to generate unique paths for each country
- Show the 'fetched' information in an appealing way
- Style project according to Behance template.
- Test the whole app to show and to behave as expected

You can watch my Loom video [here](https://www.loom.com/share/60c5d495a050483dab9fe742b37c393a)

### CODE REVIEW

File: src/components/Country.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      </div>
+    </div>
+  );
+};
+
+Country.propTypes = {
+  country: PropTypes.instanceOf(Object).isRequired,
+  id: PropTypes.string.isRequired,
+  index: PropTypes.number.isRequired,
+  // onShowData: PropTypes.func.isRequired,
```
Comment (rukundoeric) [inline src/components/Country.js:43]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/__tests__/TodayDetail.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          <TodayDetails />
+        </BrowserRouter>
+      </Provider>,
+    );
+    getByText('1.000');
+    getByText('2.000');
+    getByText('3.000');
+    getByText('4.000');
+  });
+});
```
Comment (rukundoeric) [inline src/__tests__/TodayDetail.test.js:51]: - [ ] <b>REQUIRED:</b> Your tests are failing, kindly fix them. and I really recommend adding more tests cases, you only focused on snapshots and that is not enough. Kindly add more unit tests.

Screen Shot 2022-02-04 at 00 28 27

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,26 @@
...
+  }, []);
+
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/today/:country" element={<TodayDetails />} />
+      </Routes>
+    </>
+  );
```
Comment (rukundoeric) [inline src/App.js:23]: - [ ] <b>REQUIRED:</b> Your app could not stop loading due to this error in the image below:
Screen Shot 2022-02-04 at 00 28 15
Kindly address the issue and fix it.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @williamrolando88, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 3 above)
Comment (williamrolando88) [reply src/App.js:23]: I was due to the different time zone

File: src/components/Country.js (hunk 1 above)
Comment (TedLivist) [inline src/components/Country.js:43]: - [ ] You are yet to remove the commented code as highlighted by the previous reviewer. Kindly remove it.

File: src/__tests__/Redux.test.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import {
+  fetchCountryDay,
+  fetchToday,
+  storeCountryDay,
+} from '../redux/reducers/covidData';
+import store from '../redux/storeConfig';
+
+// const dispatch = useDispatch();
```
Comment (TedLivist) [inline src/__tests__/Redux.test.js:8]: - [ ] Your new file contains commented code. Kindly remove it. It makes your codebase look messy.

File: src/__tests__/TodayDetail.test.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        <BrowserRouter>
+          <TodayDetails />
+        </BrowserRouter>
+      </Provider>,
+    );
+    getByText('1.000');
+    getByText('2.000');
+    getByText('3.000');
+    getByText('4.000');
+  });
```
Comment (TedLivist) [inline src/__tests__/TodayDetail.test.js:50]: - [ ] One of your tests is still failing, specifically this one. **NOTE:** I updated the snapshot tests as suggested in your additional info message.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @williamrolando88,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
#### _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Country.js (hunk 1 above)
Comment (williamrolando88) [reply src/components/Country.js]: I'm pretty sure I previously solved

File: src/components/HomePage.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+import LoadingScreen from './LoadingScreen';
+
+const HomePage = () => {
+  const { status, countries } = useSelector((store) => store.covidData.today);
+
+  const worldStats = {
+    todayConfirmed: 0,
+  };
+
+  /* eslint-disable */
```
Comment (rloterh) [inline src/components/HomePage.js:15]: - [ ] OPTIONAL
You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi William, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review:_
<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**